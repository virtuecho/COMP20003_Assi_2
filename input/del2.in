Buckingham
Yarraville
Parkville
Glenroy
Ascot
Buckingham
Buckingham
Buckingham
Grand Land
Kensington Gardens
Carlton
Cedar
Dream-Land
Melbourne Airport
Red Hill
Hope & Dream
Greenlands
South Windsor
East Melbourne
XX + YY = ZZ
Melbourne
Mount Richmond
Buckingham
Richmond
Back Creek
