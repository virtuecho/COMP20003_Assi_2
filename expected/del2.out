COMP20003 Code,Official Code Suburb,Official Name Suburb,Year,Official Code State,Official Name State,Official Code Local Government Area,Official Name Local Government Area,Latitude,Longitude
104,51615,Williamstown,2021,5,Western Australia,54280,Kalgoorlie-Boulder,-30.7376701,121.4863349
960,20075,Ascot Vale,2021,2,Victoria,25060,Moonee Valley,-37.7770335,144.9136945
1054,41615,Windsor,2021,4,South Australia,40150,Adelaide Plains,-34.4098434,138.3147290
1117,21327,Kensington,2021,2,Victoria,"24330, 24600, 25060","Maribyrnong, Melbourne, Moonee Valley",-37.7943466,144.9270189
1986,31507,Kensington,2021,3,Queensland,31820,Bundaberg,-24.9170988,152.3182806
2158,30567,Cedar Creek,2021,3,Queensland,"33430, 34590, 36510","Gold Coast, Logan, Scenic Rim",-27.8396390,153.1844442
2273,51399,Springfield,2021,5,Western Australia,54060,Irwin,-29.2878607,114.9540971
3201,22324,Springfield,2021,2,Victoria,21270,Buloke,-35.5747195,143.1143155
3525,12171,Kingswood,2021,1,New South Wales,16350,Penrith,-33.7644480,150.7249128
3670,10868,Cedar Creek,2021,1,New South Wales,17550,Tweed,-28.4183207,153.2623094
4278,12172,Kingswood,2021,1,New South Wales,17310,Tamworth Regional,-31.1608825,150.9283195
4338,33096,Windsor,2021,3,Queensland,31000,Brisbane,-27.4343963,153.0301686
4377,21966,North Melbourne,2021,2,Victoria,24600,Melbourne,-37.7984833,144.9448560
5262,22793,Williamstown North,2021,2,Victoria,23110,Hobsons Bay,-37.8526203,144.8689429
5388,22107,Port Melbourne,2021,2,Victoria,"23110, 24330, 24600, 25900","Hobsons Bay, Maribyrnong, Melbourne, Port Phillip",-37.8332725,144.9222073
5446,22256,Seddon,2021,2,Victoria,24330,Maribyrnong,-37.8066284,144.8917067
5496,32426,Richmond Hill,2021,3,Queensland,32310,Charters Towers,-20.0640332,146.2649959
5549,10867,Cedar Creek,2021,1,New South Wales,11720,Cessnock,-32.8372268,151.1719041
5841,13012,North Richmond,2021,1,New South Wales,13800,Hawkesbury,-33.5680371,150.7204085
6206,50752,Kensington,2021,5,Western Australia,"57840, 58510","South Perth, Victoria Park",-31.9856189,115.8839737
6363,32629,Springfield Central,2021,3,Queensland,33960,Ipswich,-27.6838705,152.9057688
6466,30568,Cedar Creek,2021,3,Queensland,35010,Moreton Bay,-27.3337142,152.8183012
6636,13640,Springfield,2021,1,New South Wales,17040,Snowy Monaro Regional,-36.5324737,149.1239824
6730,20496,Carlton North,2021,2,Victoria,"24600, 25250, 27350","Melbourne, Moreland, Yarra",-37.7865567,144.9684704
6890,22319,Spotswood,2021,2,Victoria,23110,Hobsons Bay,-37.8299577,144.8879621
7057,14343,Windsor,2021,1,New South Wales,13800,Hawkesbury,-33.6090236,150.8211213
7333,22757,West Melbourne,2021,2,Victoria,24600,Melbourne,-37.8093501,144.9251752
7502,60100,Carlton River,2021,6,Tasmania,64810,Sorell,-42.8642702,147.7112742
7720,12107,Kensington,2021,1,New South Wales,16550,Randwick,-33.9134157,151.2207450
7750,12170,Kingswood,2021,1,New South Wales,10550,Bega Valley,-36.7422084,149.8221413
8161,41325,Seddon,2021,4,South Australia,42750,Kangaroo Island,-35.8589260,137.2800867
8892,13377,Richmond Lowlands,2021,1,New South Wales,13800,Hawkesbury,-33.5789799,150.7529625
8972,22805,Windsor,2021,2,Victoria,"24600, 25900, 26350","Melbourne, Port Phillip, Stonnington",-37.8546754,144.9932064
9024,40044,Ascot Park,2021,4,South Australia,44060,Marion,-34.9881165,138.5610086
9093,20766,Docklands,2021,2,Victoria,24600,Melbourne,-37.8183004,144.9403563
9414,41602,Williamstown,2021,4,South Australia,40310,Barossa,-34.6689498,138.8960892
9726,31508,Kensington Grove,2021,3,Queensland,34580,Lockyer Valley,-27.5327853,152.4721060
9900,12357,Little Back Creek,2021,1,New South Wales,14550,Kyogle,-28.6134201,153.0458267
10486,40677,Kensington,2021,4,South Australia,45290,Norwood Payneham and St Peters,-34.9237458,138.6456183
10671,40702,Kingswood,2021,4,South Australia,44340,Mitcham,-34.9708654,138.6137673
11537,40679,Kensington Park,2021,4,South Australia,40700,Burnside,-34.9201888,138.6542865
12364,22792,Williamstown,2021,2,Victoria,23110,Hobsons Bay,-37.8611868,144.8899758
12526,22310,South Melbourne,2021,2,Victoria,25900,Port Phillip,-37.8339379,144.9572894
12972,22149,Red Hill South,2021,2,Victoria,25340,Mornington Peninsula,-38.3877331,145.0395073
12999,32630,Springfield Lakes,2021,3,Queensland,33960,Ipswich,-27.6854087,152.9178969
13321,14344,Windsor Downs,2021,1,New South Wales,13800,Hawkesbury,-33.6559424,150.8082438
13552,13378,Richmond Vale,2021,1,New South Wales,11720,Cessnock,-32.8651705,151.4971983
13842,60540,Red Hills,2021,6,Tasmania,64210,Meander Valley,-41.5290479,146.5825223
14381,22171,Richmond Plains,2021,2,Victoria,"21270, 23940","Buloke, Loddon",-36.3969563,143.4753785
14821,41571,West Richmond,2021,4,South Australia,48410,West Torrens,-34.9393884,138.5503648
15157,13376,Richmond Hill,2021,1,New South Wales,14850,Lismore,-28.7892790,153.3469865
15169,41616,Windsor Gardens,2021,4,South Australia,45890,Port Adelaide Enfield,-34.8654824,138.6543285
